core/arch/arm/plat-k3/main.c

```diff
@@ -87,3 +87,15 @@ TEE_Result tee_otp_get_hw_unique_key(struct tee_hw_unique_key *hwkey)
 
 	return TEE_SUCCESS;
 }
+
+static TEE_Result main(void)
+{
+	uint32_t swrev = 0;
+
+	if (!ti_sci_get_swrev(&swrev))
+		IMSG("Secure Software: Rev %d", swrev);
```

Comment:
s/`%d"`/`%"PRIu32`/

maybe `IMSG("TI SCI Secure Software: Rev %"PRIu32, swrev);`

Reply:
the SW revision value is written during the provisioning process (so it is not specific to TI SCI....will keep the message as Secure Software)